apps/devhub/tasks.py

```diff
@@ -144,6 +144,17 @@ def handle_file_validation_result(results, file_id, annotate=True):
     return FileValidation.from_json(file_, results)
 
 
+def addon_can_be_signed(validation):
+    """
+    Given a dict of add-on validation results, returns True if add-on can be
+    signed.
+    """
+    summary = validation.get('signing_summary', {})
+    return (summary.get('low', 0) +
```

Comment:
I think you could leave this where it was and use `result['passed_auto_validation']` for logging instead.